src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -1755,6 +1756,151 @@ class TypeSpecTest {
         |""".trimMargin())
   }
 
+  @Test fun validInlineClass() {
+    val guacamole = TypeSpec.classBuilder("Guacamole")
+        .primaryConstructor(FunSpec.constructorBuilder()
+            .addParameter("avacado", String::class)
```

Comment:
supernit: it's "avOcado" :)

Reply:
I'm horrified. I misspelled my beloved avocado. :cry:

Reply:
Lol